airflow/post_install.sh

```diff
@@ -13,11 +13,39 @@ python $AIRFLOW_HOME/utils/set_default_variables.py \
     pipeline_dataset="{{ var.value.PIPELINE_DATASET }}" \
     events_dataset="{{ var.value.EVENTS_DATASET }}" \
     source_dataset="{{ var.value.PIPELINE_DATASET }}" \
-    position_messages="position_messages_" \
-    gap_events_table="gap_events" \
-    gap_events_min_pos_count="3" \
-    gap_events_min_dist="10000" \
 
 
-echo "Installation Complete"
+for PARENT_DAG in pipe_events_daily pipe_events_monthly; do
+
+  python $AIRFLOW_HOME/utils/set_default_variables.py \
+      --force docker_image=$1 \
+      $PARENT_DAG.gaps \
+      source_table="position_messages_" \
+      events_table="gap_events" \
+      gap_min_pos_count="3" \
+      gap_min_dist="10000" \
+
+
+  python $AIRFLOW_HOME/utils/set_default_variables.py \
+      --force docker_image=$1 \
+      $PARENT_DAG.encounters \
+      source_table="encounters" \
+      events_table="encounter_events" \
```

Comment:
I would change this to `published_events_encounters`, that way there is no doubt that are `events` from this pipeline and there is no conflict. This is specially of use with `port_events`.